helm/oauth2-proxy/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: oauth2-proxy
-version: 7.15.0
+version: 7.15.1
```

As this is a new feature you need to increase the minor version not the patch

```suggestion
version: 7.16.0
```

---

Furthermore, please add a changelog entry at the bottom of the Chart.yaml for artifacthub

---

done!